components/StatisticsGoalSreak.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+const StatisticsGoalSreak = () => {
```

Comment:
StatisticsGoalSreak name needs changing to StatisticsGoalStreak 
=== spelling missing a t in streak work